cvat/apps/engine/serializers.py

```diff
@@ -305,6 +306,12 @@ def update(self, instance, validated_data):
         instance.save()
         return instance
 
+    def validate_labels(self, data):
+        label_names = [label['name'] for label in data]
+        if len(label_names) != len(set(label_names)):
```

Comment:
Could you please add one more condition to fix this issue #1054 as well?

Reply:
Sure